src/app/modules/color-scheme/color-scheme-contents/color-scheme-contents.component.html

```diff
@@ -0,0 +1,18 @@
+<div class="color-scheme-contents-container">
+  <div id="scheme-contents" class="scheme-contents">
+    <ccf-color-bar #colorbar *ngFor="scheme of schemeOptions; index as i" class="swatch"
+      [class.selected]="isSelected(i)" [class.highlighted]="gradientHighlight(i)" [colorScheme]="schemeOptions[i]"
```

Comment:
The selected/highlighted css classes belongs in the color bar component.
For the colorScheme input you have access to the scheme in the 'scheme' variable. I.e. `[colorScheme]="scheme"`